pupil_src/launchables/eye.py

```diff
@@ -290,10 +290,14 @@ def on_resize(window, w, h):
             nonlocal window_size
             nonlocal content_scale
 
-            # Always clear buffers on resize to make sure that there are no overlapping
-            # artifacts from previous frames.
-            gl_utils.glClear(gl_utils.GL_COLOR_BUFFER_BIT)
-            gl_utils.glClearColor(0, 0, 0, 1)
+            framebuffer_size = glfw.glfwGetFramebufferSize(window)
+            is_minimized = framebuffer_size is not None and 0 in framebuffer_size
```

Comment:
If the `framebuffer_size is None` we would be calling `glClear()`. I do not know if this is wanted behavior.

Reply:
The boolean expression for `is_minimized` was taken from [`pupil-invisible-monitor`](https://github.com/pupil-labs/pupil-invisible-monitor/blob/v1.3/src/pupil_invisible_monitor/window.py#L76).

Reply:
PIM uses a very different general architecture for this, you cannot just copy-paste the code literally.
In Pupil, `glfwGetFramebufferSize` always returns a tuple, so it can never be `None`.

As I wrote in [my previous comment](https://github.com/pupil-labs/pupil/pull/2009#discussion_r485403182), which @papr resolved, I liked your previous implementation of checking for positive numbers (as opposed to checking only non-zero) even more. Not sure why we dropped that.